### DISCUSSION TITLE
How to get Open Sourcerer badge ?

### DISCUSSION THREAD

**Discussion by dev-aniketj** [community/community#44972] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Please tell me how should I get the Open Sourcerer badge.
![image](https://user-images.githubusercontent.com/81229551/213864050-4f9b1f0d-03cc-4778-9d7f-df69bfc4c0a1.png)

**Comment by msuliq** [comment]
Hi @dev-aniketj 
'Open Sourcerer' achievement badge is not released to public GitHub users. You can follow any updated on the official blog of GitHub [here](https://github.blog/).

When they launched the first set of achievement badges, they previously also posted a blog over [here](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) to let users know more information. Hence, they might do the same thing (i.e. post a new blog) to announce updates to the achievement system on GitHub.

Other than that, there has been no information on how to achieve this badge.

Hope this clarifies & answers your question.

**Reply by zeus-dev907** [reply]
❤

**Reply by Daniil-l17** [reply]
❤

**Reply by brandonyee-cs** [reply]
❤

**Reply by oishylea** [reply]
❤

**Reply by lincolnvc** [reply]
❤

**Reply by ghost** [reply]
❤️

**Reply by lincolnvc** [reply]
❤️

**Reply by CxrlosKenobi** [reply]
❤️

**Reply by jos556** [reply]
❤️

**Reply by dev-x-master** [reply]
❤️

**Reply by surajgour-d11** [reply]
❤️

**Reply by jatixs** [reply]
❤️

**Reply by viniciusvts** [reply]
❤️

**Reply by itsKiiro** [reply]
❤️

**Reply by anand-designer** [reply]
❤️

**Reply by borjantona** [reply]
❤️

**Reply by martink1337** [reply]
❤️

**Reply by tejashinde** [reply]
❤️

**Reply by peachmontt** [reply]
❤️

**Reply by Mr-the-beginner** [reply]
I know the solution, look at my profile i have them both
Heart on your Sleeves => you have to react to the second reaction to someones second reply on an open issue , a heart emoji, for the Bronze it should be 16th , i dont know about the silver and gold

Open Sourcerer => you have to get 2 pull request merged into a repo with more than 900 stars , for its bronze version it should be 8 pull requests merged, for the silver 16 and gold 64

also check this two github profiles and look at the badges =  
https://github.com/CC11001100
https://github.com/Mr-the-beginner

**Reply by SheepChef** [reply]
> I know the solution, look at my profile i have them both Heart on your Sleeves => you have to react to the second reaction to someones second reply on an open issue , a heart emoji, for the Bronze it should be 16th , i dont know about the silver and gold
> 
> Open Sourcerer => you have to get 2 pull request merged into a repo with more than 900 stars , for its bronze version it should be 8 pull requests merged, for the silver 16 and gold 64
> 
> also check this two github profiles and look at the badges = https://github.com/CC11001100 https://github.com/Mr-the-beginner

They are probably testing, I got both of these and they currently dissappeared.

**Reply by Mr-the-beginner** [reply]
yes mine got deleted too

**Comment by Tynab** [comment]


**Reply by msuliq** [reply]
I have contributed to an open source Ruby gem 2 months ago and have not received this achievement. Is there any specific criteria from GitHub as what counts an "open source project"? Could you please share a link to where have you found this information? :)

**Reply by xamidi** [reply]
@msuliq The "source" of that misinformed spam post is clearly AI. If you have trouble detecting AI-generated garbage, you can use tools like https://quillbot.com/ai-content-detector.

**Reply by anand-designer** [reply]
❤️ Nice

**Reply by peachmontt** [reply]
❤️

**Reply by SafaMaqbool** [reply]
i got this badge today but it showed that it was opened on 27th dec when i did my first open source contribution which was merged successfully and now suddenly the badge dissapeared 😭 whats the issue?

**Comment by YasinDehfuli** [comment]
Open sourcer is a Unreleased badge that you cant get it now, 
but if you need help to get other achievements, you can use this repository: 

https://github.com/4xmen/Get-Github-Achievements

it may help you to get any achievement you want step by step

**Comment by Peshala84** [comment]
you can get some details about badges  using this readme file  [https://github.com/drknzz/GitHub-Achievements.git](url)

**Reply by iufor** [reply]
Hello
 I have same issue  facing 
 i find a solution

**Reply by Mr-the-beginner** [reply]
I know the solution, look at my profile i have them both
Heart on your Sleeves => you have to react to the second reaction to someones second reply on an open issue , a heart emoji, for the Bronze it should be 16th , i dont know about the silver and gold

Open Sourcerer => you have to get 2 pull request merged into a repo with more than 900 stars , for its bronze version it should be 8 pull requests merged, for the silver 16 and gold 64

also check this two github profiles and look at the badges =
https://github.com/CC11001100
https://github.com/Mr-the-beginner

**Reply by ljharb** [reply]
they must have added this recently; i never used to have either of those badges, but now i have both - backdated to 2016 and 2012.

**Reply by Mr-the-beginner** [reply]
damn bro , you have all the achievments

**Reply by ljharb** [reply]
looks like they're gone again. (also open sourcerer was identical to pull shark in terms of criteria)

**Reply by Mr-the-beginner** [reply]
what , whyyyyyy

**Reply by Mr-the-beginner** [reply]
https://github.com/orgs/community/discussions/190842

just read these :))) happend to everyone

**Comment by tony9823** [comment]
How can I earn the Open Sourcerer badge on GitHub? Are there specific contributions, repositories, or milestones [here](https://minahilfreshsimdetails.com/) I need to achieve to qualify for this recognition?

**Comment by modesto15** [comment]
All app issue update my gcash account activity

**Comment by cryptoking-max** [comment]
All app issue update my gcash account activity

**Comment by openSourcerer9000** [comment]
It's not hard, just send me your application fee and I'll consider whether to bestow my magic upon you. 🧙

**Comment by SafaMaqbool** [comment]
That’s unfair 😭😭😭

**Comment by samus-aran** [comment, marked as answer]
We apologize for the temporary re-instatement of the **Heart On Your Sleeve** and **Open Sourcerer** achievements. This was done in error - those achievements were part of an experimental rollout and weren’t meant to be re-enabled broadly.

Once we identified the mistake, we removed them again.
